auth-web/src/stores/business.ts

```diff
@@ -484,7 +484,10 @@ export const useBusinessStore = defineStore('business', () => {
 
   async function removeBusiness (payload: RemoveBusinessPayload) {
     // If the business is a new registration then remove the business filing from legal-db
-    if (payload.business.corpType.code === CorpTypes.INCORPORATION_APPLICATION) {
+    if ([CorpTypes.AMALGAMATION_APPLICATION,
+      CorpTypes.INCORPORATION_APPLICATION,
+      CorpTypes.CONTINUATION_IN,
+      CorpTypes.REGISTRATION].includes(payload.business.corpType.code)) {
```

Comment:
We might have orphan draft registration filing in filings table which was unaffiliated from the account.

Reply:
And orphan draft amalgamation applications and firm registrations?

Please call this out to Riyaz for extra testing.

Reply:
since amalgamation is not enabled in prod it wont be there

Reply:
OK, so there is no bug in Prod, but we might  see it in Dev or Test (and later Prod when we enable it there).

I agree with your code change; I just want to understand its impact -- what does this fix?

Reply:
there is a bug in prod for firm registration. I don't think there will be an impact though (its going to stay there for ever). Its hard to find the orphan drafts since the link between draft and account in different db (draft filing in lear and affiliation in auth)